assets/js/base/components/product-list/no-matching-products.tsx

```diff
@@ -5,7 +5,14 @@ import { __ } from '@wordpress/i18n';
 import { useInnerBlockLayoutContext } from '@woocommerce/shared-context';
 import { Icon, search } from '@woocommerce/icons';
 
-const NoMatchingProducts = ( { resetCallback = () => {} } ) => {
+/**
+ * Internal dependencies
+ */
+import { NoMatchingProductsProps } from './types';
+
+const NoMatchingProducts = ( {
+	resetCallback = () => void 0,
```

Comment:
Noob question, what is 0 in the `void 0` for?

Reply:
`void` is an operator in JS, which means it needs an expression after it. Much like you can't just write `const` without defining a value after it. You can read up about it more [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/void) if you want. There was also a [discussion on slack](p1638213487071400-slack-C8X6Q7XQU) about the standard way to type empty functions and we decided to use `() => void 0`